### DISCUSSION TITLE
Clouds too bright in cam6_3_107

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#253] [TF2: Tuning]
From @tilmes: 
Thanks for the discussions today. For the discussion today, I think it may be important to look at the following questions:
- what is the current sulfate loading in the model (how does it compare to observations)
- If it is too high, are DMS emissions too high and can they be reduced
- If it OK, how much sulfate forms clouds. So in particular what clouds are actually impacted, are these mostly liquid clouds, or ice clouds, can we change the amount of sulfate that form CCN?
For the first part, could you point me to a run that may be the "best run", and I can look at sulfate burden and distribution? I may perform another run over the Atom period (2016-2018) to I can compare to aircraft observations.
Second, can someone look at CNN and compare to a previous run, so we could see where they are increasing, maybe that helps us track down the problem area.

@adamrher, @brianpm, @andrewgettelman, @JulioTBacmeister, @cecilehannay, @bstephens82

**Comment by cecilehannay** [comment]
From Adam: that'd be great if you could take a look at the DMS emissions and sulfate burdens/lifte-time/etc ... Cecile, pls correct me if I'm wrong, but I believe the run she should analyze is this one:

/glade/scratch/hannay/archive/f.cam6_3_105.FLTHIST_v0a.ne30.001

**Comment by brianpm** [comment]
I used the 2000-2008 climatology of this run to compare to the CALIPSO (2006-2021) climatology and the CAM6 AMIP run (1999-2012). Maps of total cloud cover below.

As I mentioned, and has been true in every version of CAM, there is a huge underestimation of cloud amount across the tropical oceans in the low-cloud regions. 

This run looks pretty different in that it is overestimating cloud cover in many convective regions and notably also in the adjacent regions like the Gulf of Guinea and northern Australia.  I'd also point out that the Southern Ocean is a region where CAM actually has too much cloud (and it is mainly because of too much low-level cloud), and that bias seems slightly worse in this run.



![image](https://user-images.githubusercontent.com/5471157/233722501-d44cd2c3-fa5b-4cf5-a881-5774c1392aba.png)

**Reply by tilmes** [reply]
@brianpm would it make sense to continue the run to 2015, then we could look at the same period between CALIPSO and CAM6 and CAMdev. Aerosol emissions are changing a bit in the last 20 years, that is why I am suggesting it.
@cecilehannay would it be easy to continue the run (and add some more aerosol variables that I sent you)?

**Reply by adamrher** [reply]
The tropical clouds look overall improved compared to CAM6 (pink convection biases not withstanding). My hypothesis has been that the inversion layer is better resolved in L58, and mitigates erroneous cloud top entrainment events.

I don't like the worsening bias over SO and Antartica. My colleague Rene looked at 099 tag and Antarctica is much colder, which worsens an already cold bias. I am still dcs=250E-6 curious, as that seems to resolve are issues in that region.

**Reply by brianpm** [reply]
@tilmes -- I doubt that extending the run would change this basic result. I'd hold off unless we think we are ready to dive in and look at seasonal signals and interannual variability. 

@adamrher -- I agree, this run actually looks overall better than standard CAM6, at least from this first look. I'm concerned about that pink stripe across the Pacific as much as anything else that we see here.

**Reply by cecilehannay** [reply]
@tilmes: I was going to add the new variables in the next run. 
```
'so4_a5', 'num_a5', 'SFSO2', 'OCN_FLUX_DMS', 'SAD_SULFC', 'SAD_TROP', 'SAD_AERO'
```

**Reply by tilmes** [reply]
@brianpm SO2 is changing a bit between the different years, it has a trend that goes up and down. If we are worried about aerosols, I think, doing the same years would be best especially in comparison to observations. I am thinking of cloning Cecile's code and run this from 2000-2020, and I also use different emissions that go all the way to 2020. I can add the same diagnostics for you to be able to compare to the clouds if you are OK looking at it.

**Reply by adamrher** [reply]
@brianpm to piggy-back on the issue with the pink stripe. @shivpriyamr provided a vertical transect [here](https://github.com/NCAR/amwg_dev/discussions/247#discussioncomment-5642324). That is from a CESM2 AMIP run. This suggests it was bad then, and it's worse now.

dcs=250E-6 may help us here as well.

**Reply by andrewgettelman** [reply]
CCN3 and column CDNUMC would be the cloud related variables to look at. S. Ocean we probably want more ice nucleation (not exactly sure how to get that, would have to think a bit).

We also discussed that the land surface (clear sky SW?) looked like it had a big bias, and if I recall correctly it wasn't bright enough, and we could see about brightening it. That would (A) cool the planet reducing RESTOM and (B) more negative FSNTC would reduce the SWCF automagically (less difference from surface).

That does not absolve us from the other issues. One metric we have used in the past is the spatial RMSE of annual mean SWCF v. CERES-EBAF. That might be useful to see where we are (from Brian's plot we might be in better shape than CESM2-CAM6)

**Reply by adamrher** [reply]
> We also discussed that the land surface (clear sky SW?) looked like it had a big bias

Indeed it appears we've gotten lucky here, as the new land model runs seems to have [reduced this bias substantially](https://github.com/NCAR/amwg_dev/issues/250#issuecomment-1518326661).

**Reply by tilmes** [reply]
@cecilehannay I tried to run with different emissions (that go till 2020) but there are other places I would need to add changes.
Therefore, I wonder if you could just continue this run until 2015, so I can compare the results with more recent years?

**Comment by JulioTBacmeister** [comment]
I like many aspects of the DCS=250 run especially in the Southern Ocean. We can't live with LWCF<20, but  I'd be interested in doing more exploration with DCS=250.

@brianpm is the too-few-too-bright worse when DCS is lower?

**Reply by adamrher** [reply]
I would suggest a run with DCS=250. and a small'ish reduction in clubb_detice_rad=61->(30-40) microns. Based on Ben's prior run, where he reduced it to down to 1 microns, we may expect it to both increase LWCF and SWCF together. But we don't know the behavior of a smaller perturbation to 30-40 microns.

Our cold bias over Antarctica and the Arctic land masses, relative to ERA5, got a lot worse in the 099 tag. So that's part of my motivation for wanting to retain the clouds with DCS=250.

**Reply by JulioTBacmeister** [reply]
Is there a physical argument to justify ANY particular value of DCS?? Is there any reason not to let it vary with background conditions?

**Reply by dan800** [reply]
@adamrher I'd be a little wary of tuning physics to avoid issues with dynamics in LT when the target configuration is the workhorse MT. At some point we just need to bite the bullet and start tuning in a model config with a resolved stratosphere where we can target biases in polar regions with GW tuning.

**Reply by adamrher** [reply]
I hear ya. I'm all about starting coupled ASAP, to see what things look like. I expect the coupled to increase SWCF, as it is able to cool SSTs and thicken clouds further. And point taken, that the GW tuning may indeed help with the high-latitude temperature biases I going on about. But I don't expect the clouds to be much different between LT and MT. 

The clouds need some work, what can I say? The co-chairs mtg agreed to give us until next Tuesday to get our F-case in order, so why not try some more things?

**Reply by bstephens82** [reply]
Just FYI I believe the run Adam is referring to is one where I used 10 microns (not 1 micron, since Andrew said he had previously used 2-10 microns I think). I also have results for 30 and 40 microns I believe. I could put those online (might be tomorrow, I have family in town right now) if people are interested.

**Reply by dan800** [reply]
> I hear ya....And point taken, that the GW tuning may indeed help with the high-latitude temperature biases I going on about. But I don't expect the clouds to be much different between LT and MT.

Presumably clouds respond to tropospheric temperature / pressure changes and reducing biases in those by taming the stratospheric jets and improving the annular modes should be a better starting point from which to then tune cloud physics. I worry you're trying to tune out a large scale dynamical bias with the wrong set of knob twiddles.

**Reply by JulioTBacmeister** [reply]
We'll go ahead and see how the next F-case with the more optimal cloud tuning from Ben. We should know soon.  I agree that it is important to start an L93 F-case run ASAP. I think that is what Dan is advocating. However, we need a compset for L93 first.

**Reply by brianpm** [reply]
RE: too-few-too-bright: I don't know. I can take a look when I get back next week if Cecile/Ben still has the COSP output for runs with different DCS.

**Reply by cecilehannay** [reply]
I have the COSP output in the most recent runs. It includes the dcs runs.

**Reply by brianpm** [reply]
As a summary, this bar plot shows the error in the climatology of total cloud cover (CLDTOT_CAL) for recent runs. 

The blue is the unconditional bias, basically the error in the global mean (actually 65S-65N). The green is the phase error, which is basically the pattern correlation. The orange is the conditional bias, which is a combination of amplitude error and phase error. Lower is better. The "reference case" is the CESM2-CAM6 AMIP run (member 10; 2000-2014). The observations are CALIPSO, 2006-2020. 

I just looped over recent cases in @cecilehannay's archive directory. Some of them are very short, and that probably explains the one outlier (one month of output). 

I can't do this for @bstephens82's recent run (`107.clm51_vtrmi0.5_dcs250_wsubi2.5_seasalt2.0`) because it doesn't have COSP output.

![camdev_multicase_cldtot_cal_nmse](https://user-images.githubusercontent.com/5471157/236050763-a9540e65-7edd-4fa5-9074-eb21edf51c41.png)

**Comment by tilmes** [comment]
@andrewgettelman and I have discussed potentially including accumulation mode sulfates to increase the ice nucleation over the Southern Ocean. We could try this change if that would be helpful? I checked the sulfate budget and it does not look out of the order in comparison to earlier CESM2 version.

**Comment by tilmes** [comment]
@JulioTBacmeister You said, you also wanted to test this to test it if changes the LWCF?
change hetfrz_dust_scalfac=0.05 to hetfrz_dust_scalfac 0.1 or just as a test to 0.5 and see what happens.

**Reply by andrewgettelman** [reply]
I'm still a bit lost on what the runs are looking like. Could someone repost a link to diagnostics for the latest DCS = 250 runs we have been looking at?

In response to @JulioTBacmeister comment about DCS: it's a physical size (diameter) beyond which ice crystals become snow and start falling faster. So this can be observed, but it does vary. We typically have put this higher than we wanted to try to keep more ice (i.e. 500 microns). So 250 or so is fine. Much below 100 doesn't make much physical sense (there does tend to be a lot of ice in that size).

**Reply by JulioTBacmeister** [reply]
@tilmes thanks for these ideas.  I have a rough sense for what the first suggestion is meant to do physically (increase ice content via a sulfate change) and I think it sounds worth trying.  I don't have any sense for how this is done in the code. Please let us know.

I'm afraid I'm a bit fuzzy on the second suggestion - increase hetfrz_dust_scalfac, but I'm guessing it is also meant increase ice cloud by putting more nuclei in the atmosphere.  Let's try that as well.  Let's talk to @cecilehannay unless you are able to do the runs.

**Reply by JulioTBacmeister** [reply]
@andrewgettelman the DCS tuning have been carried out mostly by Ben Stephens. He put a table with links to diags here: https://docs.google.com/spreadsheets/d/1fYeWnCmUJAPWKxJD56te0aHpA8azVjMV91wOvBnlvS0/edit?usp=sharing

Hover over the text in the first row and you'll see the links

**Reply by JulioTBacmeister** [reply]
@tilmes Ben (@bstephens82 ) will start a run with hetfrz_dust_scalfac increased.

**Reply by tilmes** [reply]
OK great!  @bstephens82  Ca you just set it to 0.5 and see if it it has a big effect? If so we can scale it down. 
For the other experiment, I will provide some new code.

**Reply by bstephens82** [reply]
@JulioTBacmeister @tilmes I started a run earlier today with hetfrz_dust_scalfac=0.5 and I have the diagnostics...I don't think I can put them online right now due to the current Mesa Lab issues, but here are the cloud forcings.  Relative to the simple dcs=250 run, this brings the global mean LWCF up to ~20.7 and dims the mean SWCF a bit.  For the SWCF, it looks like the stratus decks might not be quite as good and some of the southern ocean brightness has come back I think.  I will post the full diagnostics as soon as the computers are back up.

![set5_ANN_LWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/234379187-c717b996-1aa5-479e-bb67-f52e87906efd.png)


![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/234379143-00aa4b8c-9dac-42db-8c1f-7d1c5b122427.png)

**Comment by tilmes** [comment]
@cecilehannay sorry, just noticing that so4_a5 is not in CAM (low top GHG chemistry), so you can remove it again from the namelist.

**Comment by cecilehannay** [comment]
Yes, the model was crashing. Indeed, I removed both: ``'so4_a5', 'num_a5'`` and then it ran fine.

**Comment by bstephens82** [comment]
I was able to get a pretty decent looking set of cloud forcings and a more reasonable RESTOM in the 107 tag with the following namelist settings:

micro_mg_dcs=400e-6 (down from 500e-6)
micro_mg_autocon_lwp_exp=2.4 (down from 2.47)
micro_mg_vtrmi_factor=0.5 (down from 1.0)
seasalt_emis_scale=2.0 (up from 1.0)

This setup gives a RESTOM around 1.8, a SWCF ~ -46.3, and a LWCF ~ 22.8.  The stratus decks look not bad to me, although the southern ocean is still a bit too bright.  @tilmes is seasalt_emis_scale=2.0 a reasonable value, or is that too large?  I notice that in some configurations it has a default value as high as 1.62.

The stats for this run (and a similar run with the 099 tag with similar results) are available on my spreadsheet [here](https://docs.google.com/spreadsheets/d/1fYeWnCmUJAPWKxJD56te0aHpA8azVjMV91wOvBnlvS0/edit?usp=sharing).

![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/234886352-f17dc927-9d4b-4fa6-9c70-e1b0bfb15d93.png)

**Reply by tilmes** [reply]
@bstephens82 I think, it is fine to reduce the seasalt_emis_scale to smaller values and see if that improves the clouds in the  Souther  Ocean.

**Reply by andrewgettelman** [reply]
That does look decent. But the LWCF is still a bit low. I am a bit concerned it's open season on autoconversion and accretion if we are suddenly modifying exponents all over the place. Is there a justification for changing the KK2000 LWP exponent that has been discussed? That might have some unintended consequences on feedbacks. A small change might be okay in this case, but if it has a large effect, we might need to justify it. 

I think for starting coupled tuning it's probably fine.

**Reply by bstephens82** [reply]
@andrewgettelman I think I assumed we had some leeway on the parameters in the [figure Julio posted](https://github.com/NCAR/amwg_dev/discussions/247#discussioncomment-5618919), which includes autocon_lwp_exp, and I've been exploring the ones that the figure indicates might be useful for tuning clouds forcing and RESTOM, but of course I want to keep parameters within acceptable ranges.  If 2.4 is too small for this parameter or if I should just leave it at 2.47, just let me know.  I'm exploring various ways of getting better clouds forcing etc.

@tilmes According to PPE results, a larger value of seasalt_emis_scale can bring down RESTOM, and I was exploring ways of bringing that down when I got this result, so I was wondering if increasing this parameter is okay.  It looks like it has larger values in some configurations (up to 1.62), but do you think 1.5-2.0 range is too high for cam_dev L58?

**Reply by dan800** [reply]
> I think for starting coupled tuning it's probably fine.

If this is *the* version to start coupled tuning we should be taking a look at the general morphology as well. What are the non-orographic gravity wave set to in this version? Are they turned on? Since microphysics has changed, what has happened to the water vapor entry value in the stratosphere in this version?

**Reply by tilmes** [reply]
@bstephens82 If you increase Sea-salt to much, you may get brighter clouds I would think (increasing the RESTOM), but the direct effect of sea-salt may give you a smaller RESTOM.For sea-salt, the direct effect can be  stronger depending on the size of the ses-salt. If you look back at your sea-salt test runs, did you see the brightness of clouds in the Southern ocean change with different amount of sea-salt emissions?

**Reply by bstephens82** [reply]
@tilmes Yes increasing seasalt_emis_scale from 1.0 to 2.0 does brighten things in the southern ocean a bit (SWCF goes from roughly -45.8 to -46.3), but brings RESTOM down from ~3.6 to ~1.8.

**Reply by andrewgettelman** [reply]
> @andrewgettelman I think I assumed we had some leeway on the parameters in the [figure Julio posted](https://github.com/NCAR/amwg_dev/discussions/247#discussioncomment-5618919), which includes autocon_lwp_exp, and I've been exploring the ones that the figure indicates might be useful for tuning clouds forcing and RESTOM, but of course I want to keep parameters within acceptable ranges. If 2.4 is too small for this parameter or if I should just leave it at 2.47, just let me know. I'm exploring various ways of getting better clouds forcing etc.

Hi @bstephens82 , the issue is that not everything that is a namelist variable SHOULD be modified without justification. The autoconversion and accretion are equations based on fits to cloud resolving model data and observations. We have not used them for tuning in the past, and doing so needs to be done more intentionally. There is of course no way of knowing this really a priori from looking at a namelist. If @JulioTBacmeister wants to go down this route, that's fine. But this equation controls a lot of things, so divorcing it from a documented value has some risks. When we changed the autoconversion Nd exponent for CAM6, there was a lot of analysis and justification.

**Comment by dan800** [comment]
Is #253 the version @andrewgettelman is proposing to use for coupled tuning? I can't seem to find #253 in the issues tab. I see 252, 254 (closed) and 255 - can someone point me to it? Without looking at the namelist, I have no idea if this has non-orographic GWs on or off, or what tau_0_ubc is set to.

**Reply by bstephens82** [reply]
Hi @dan800 sorry for the delay.  The run that I was referring to above (which @andrewgettelman said might be okay to use to start coupled runs) has output located at 

/glade/scratch/stepheba/archive/107_vtrmi0.5_lwpexp2.4_dcs400_seasalt2.0

and the run and case directories are

run: /glade/scratch/stepheba/107_vtrmi0.5_lwpexp2.4_dcs400_seasalt2.0/run

case: /glade/u/home/stepheba/cam107/cime/scripts/107_vtrmi0.5_lwpexp2.4_dcs400_seasalt2.0.

After speaking with @JulioTBacmeister, I think non-orographic gravity waves are turned off, and you could hopefully use the monthly output to extract the water vapor entry value for the stratosphere.

**Comment by bstephens82** [comment]
I have a run now with RESTOM ~ 1.1, SWCF ~ -46.8, and LWCF ~ 23.5.  The stratus decks look decent to me, maybe actually a bit too bright off South America.  And the southern ocean is a little bright.  But overall it might be decent.  It also does not make use of micro_mg_autocon_lwp_exp since @andrewgettelman was concerned about that.  The parameters used for this case are:
- micro_mg_dcs = 250e-6
- micro_mg_vtrmi_factor=0.5
- microp_aero_wsubi_scale=2.5
- seasalt_emis_scale=2.0

Also note this is using the 107 tag but with the updated land model (CLM51, ctsm5.1.dev122).
https://webext.cgd.ucar.edu/F2000climo/107.clm51_vtrmi0.5_dcs250_wsubi2.5_seasalt2.0/atm/107.clm51_vtrmi0.5_dcs250_wsubi2.5_seasalt2.0-obs.0001_1/

**Reply by bstephens82** [reply]
Here are the 10-year diagnostics for the run with the parameters listed above:
https://webext.cgd.ucar.edu/F2000climo/107.clm51_vtrmi0.5_dcs250_wsubi2.5_seasalt2.0/atm/107.clm51_vtrmi0.5_dcs250_wsubi2.5_seasalt2.0-obs.0001_10/

**Reply by andrewgettelman** [reply]
From a global perspective that looks pretty good. The clouds seem 'decent'. AODVIS I think is a bit high (probably the sea salt scaling), but it could work as a start.

**Reply by adamrher** [reply]
Agreed, this looks good. My understanding is that for a higher sea salt emission factor, it reduces the sea salt emissions. I thought that's what Simone said.

I took a look at how this run compares to a "control" ... Ben I took the run "107.clm51" to be the control. There's only 1 year of that control, so here are some 1 year differences of T and Q:

![temp_dzonal_Q](https://user-images.githubusercontent.com/29961476/236035723-b6ae6ee6-1e0f-4c05-970b-8049a14572d7.png)

![temp_dzonal_T](https://user-images.githubusercontent.com/29961476/236035770-180cc690-13f8-413e-b021-3410f7dbafb1.png)

**Reply by dan800** [reply]
So this version has 30% more water vapor in the tropical lower stratosphere - is that right?

**Reply by adamrher** [reply]
Yes, that's what it shows. I would've expected that based on prior experience lowering the value of ```micro_mg_vtrmi_factor```, which proportionally reduces cldice and snow fall speeds, but I was not expecting the warming of the upper troposphere. It actually raises the tropopause a bit:

![temp_dzonal_T](https://user-images.githubusercontent.com/29961476/236046908-8e089b3f-afb0-48ee-9ab3-f02acaaaeec8.png)

I think we probably need a longer control to determine how robust this result is.

**Comment by dan800** [comment]
Running several years to look at the tropical tape recorder would be good.

On Wed, May 3, 2023 at 2:54 PM Adam Herrington ***@***.***>
wrote:

> Yes, that's what it shows. I would've expected that based on prior
> experience lowering the value of micro_mg_vtrmi_factor, which
> proportionally reduces cldice and snow fall speeds, but I was not expecting
> the warming of the upper troposphere. It actually raises the tropopause a
> bit:
>
> [image: temp_dzonal_T]
> <https://user-images.githubusercontent.com/29961476/236046908-8e089b3f-afb0-48ee-9ab3-f02acaaaeec8.png>
>
> I think we probably need a longer control to determine how robust this
> result is.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/amwg_dev/discussions/253#discussioncomment-5796662>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJBT7IHH5HXTUNHTFWVFWRTXELAWXANCNFSM6AAAAAAXHHUG7Y>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Climate and Global Dynamics Laboratory & The High Altitude Observatory
National Center for Atmospheric Research, Boulder, Colorado 80307-3000
phone: +1 303 497-1160     email: ***@***.***      github.com/dan800
